examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -110,7 +110,7 @@ spec:
         {{- end }}
         {{- if .Values.extraEnv }}
         env:
-          {{- toYaml .Values.extraEnv | nindent 10 }}
+          {{- toYaml .Values.extraEnv | nindent 8 }}
```

Comment:
Was this an unrelated bug?

Reply:
A semi-related formatting issue. I added `SSL_CERT_FILE` to the environment as part of this PR and it turned out that we were over-indenting anything else passed via `extraEnv`.